arcgis-ios-sdk-samples.xcodeproj/xcshareddata/xcschemes/arcgis-ios-sdk-samples.xcscheme

```diff
@@ -1,10 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
    LastUpgradeVersion = "1200"
-   version = "1.3">
+   version = "1.7">
    <BuildAction
       parallelizeBuildables = "YES"
       buildImplicitDependencies = "YES">
+      <PreActions>
+         <ExecutionAction
+            ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
+            <ActionContent
+               title = "Download Portal Item Data"
+               scriptText = "/usr/bin/osascript -e &apos;display notification &quot;Build will start after download is complete&quot; with title &quot;Downloading Portal Data&quot; subtitle &quot;Prebuild&quot; sound name &quot;Funk&quot;&apos;&#10;&#10;PORTAL_ITEMS=&quot;${SRCROOT}/arcgis-ios-sdk-samples/Content Display Logic/PortalItems.plist&quot;&#10;FILE_TYPES=&quot;${SRCROOT}/arcgis-ios-sdk-samples/Content Display Logic/FileTypes.plist&quot;&#10;DOWNLOAD_DIRECTORY=&quot;${SRCROOT}/Portal Data&quot;&#10;xcrun --sdk macosx swift &quot;${SRCROOT}/Scripts/download-portal-item-data.swift&quot; &quot;$PORTAL_ITEMS&quot; &quot;$FILE_TYPES&quot; &quot;$DOWNLOAD_DIRECTORY&quot;&#10;&#10;/usr/bin/osascript -e &apos;display notification &quot;Project will start to build&quot; with title &quot;Download Complete&quot; subtitle &quot;Prebuild&quot; sound name &quot;Glass&quot;&apos;&#10;">
```

Comment:
This is the same script from the build phase, with 2 notifications added to indicate that the pre-action is in progress.

However, during the download process, user won't get additional information and may think the build process halt or run into problems. Probably we can instead send notifications when each file completes?